src/components/site/PageHeader.js

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import PropTypes from "prop-types";
+import classnames from "classnames";
+
+const PageHeader = props => {
```

Comment:
This component needs to be exported in order to work with the prototype. The export file is located here `src/index.js`